vagrant/boot_cluster.sh

```diff
@@ -21,21 +21,20 @@ for i in 1 2 3 4 5; do
 done
 ps auxww | grep -i kafka
 
-N=10
+N=120
 RC=1
 set +x
-printf "Waiting for Kafka to become available."
+printf "Waiting for kafka5 to become available."
```

Comment:
Is the 5 intended or a typo?

Reply:
Intended, the `nc` poll in this block is only querying the fifth broker in the cluster, known as "kafka5" in the toxiproxy config and toxi port 909`5` --> kafka port 2909`5`